### ISSUE TITLE
Accessibility Report: Link Component

### ISSUE THREAD

**Issue by sherakama** [SU-SWS/decanter#320]
Use the AMP tool to scan the [Link](http://decanter.stanford.edu/section-components.html#kssref-components-link) component for accessibility issues and document them to this ticket.

**Comment by sherakama** [comment]
AMP Report: https://stanford.levelaccess.net/public/reporting/view_report.php?report_id=283426

No violations found.

**Comment by sherakama** [comment]
Chrome Audit: 100%

![Screen Shot 2019-03-13 at 9 04 05 PM](https://user-images.githubusercontent.com/550602/54330589-9fa02780-45d3-11e9-852b-fdad509645a4.png)
